app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseInterface.kt

```diff
@@ -410,6 +387,8 @@ interface DatabaseInterface {
      */
     fun getAvailableItems(): Map<String, Int>
 
+     */
+
     /**
      * Update, or add if it was not already in the database, the current location
      * (provided by the GeoPoint) of the user in the database.
```

Comment:
If commented, then useless, then you could remove it

Reply:
thanks, removed in last commit !